src/toast/ops/noise_estimation.py

```diff
@@ -173,7 +173,7 @@ class NoiseEstim(Operator):
     )
 
     pairs = List(
-        default_value=None,
+        # default_value=None,
```

Comment:
Thanks, I will fix all of these in a systematic way during the fixes to the template calibration.